models/gc_spider_baseline/utils/SpiderBaselineRunner.py

```diff
@@ -0,0 +1,155 @@
+"""
+-------------------------------------------------------------
+Mhub / DIAG - Run Module for the GC Spider baseline Algorithm
+-------------------------------------------------------------
+
+-------------------------------------------------------------
+Author: Sil van de Leemput
+Email:  sil.vandeleemput@radboudumc.nl
+-------------------------------------------------------------
+"""
+from pathlib import Path
+import shutil
+import os
+import json
+
+import SimpleITK
+import numpy as np
+
+from mhubio.core import Instance, InstanceData, IO, Module
+
+# The GC Spider baseline algorithm is imported as a self-contained algorithm class with an execute method
+from TestSpine import Module as SpiderAlgorithm
+
+
+@IO.Config('traversal_direction_up', bool, True, the="Direction to traverse the image")
+@IO.ConfigInput('in_data', 'nifti|nrrd|mha:mod=ct|mr', the='supported datatypes for the spider baseline model')
+class SpiderBaselineRunner(Module):
+
+    traversal_direction_up: bool
+
+    SPIDER_DATA_DIR = Path(os.environ["VERSEG_BASEDIR"])
```

Comment:
Why don't we set these as file globals so that we can omit the `self.` ?
They're not MHub-specific and can improve readability.

Reply:
That's fine, I'll do that.